run.py

```diff
@@ -1,4 +1,33 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+
 from app import app
 
-if __name__ == '__main__':
-    app.run(host='0.0.0.0')
+if __name__ == "__main__":
+    parser = ArgumentParser(
+        description="Run a web server to analyze Python applications and report results."
+    )
+    parser.add_argument(
+        "--host", "-H", type=str, default="127.0.0.1", help="The interface to bind to."
+    )
+    parser.add_argument(
+        "--port", "-p", type=int, default=5000, help="The port to bind to."
+    )
+    parser.add_argument(
+        "--debug",
+        "--debugger",
+        "-d",
+        action="store_true",
+        default=False,
+        help="Enable the debugger.",
+    )
+    parser.add_argument(
+        "--threaded",
+        "--with-threads",
```

Comment:
Here too, `--threaded` and `-t` is enough I think.

Reply:
OK, I'll change it. Just FYI, these are `--with-threads` and `--debugger` are from the `flask run` CLI.

```
[xhardy@IT-IRL00950 pyreportcard (issue-13)]$ flask run --help
Usage: flask run [OPTIONS]

  Run a local development server.

  This server is for development purposes only. It does not provide the
  stability, security, or performance of production WSGI servers.

  The reloader and debugger are enabled by default if FLASK_ENV=development
  or FLASK_DEBUG=1.

Options:
  -h, --host TEXT                 The interface to bind to.
  -p, --port INTEGER              The port to bind to.
  --cert PATH                     Specify a certificate file to use HTTPS.
  --key FILE                      The key file to use when specifying a
                                  certificate.
  --reload / --no-reload          Enable or disable the reloader. By default
                                  the reloader is active if debug is enabled.
  --debugger / --no-debugger      Enable or disable the debugger. By default
                                  the debugger is active if debug is enabled.
  --eager-loading / --lazy-loader
                                  Enable or disable eager loading. By default
                                  eager loading is enabled if the reloader is
                                  disabled.
  --with-threads / --without-threads
                                  Enable or disable multithreading.
  --help                          Show this message and exit.
```

Reply:
Oh, I did not recognize that `--debugger` and `--with-threads` is from `flask run`. So .. in this case, using them might be better? How do you think about this?

Reply:
As you wish, I don't have any strong opinion about that.

Reply:
Then, use `-d`/`--debugger` and `-t`/`--with-threads` for each. Sorry for the confusion caused to you.

Reply:
What about `--debug` and `--threads`?

Reply:
That's ok too. I just want to use only one long and one short option, not more than 2.

Reply:
OK, as I've already changed it to `--debug` and `--threaded`, let's not change it again. Thanks for your feedback.